### DISCUSSION TITLE
Source code of India Localisation

### DISCUSSION THREAD

**Discussion by mdkhemka** [microsoft/ALAppExtensions#12215] [Q&A]
https://github.com/microsoft/ALAppExtensions/tree/master/Apps/IN

Is this source code latest? I feel some codes are not available. Please help me to get full source code.

**Comment by JesperSchulz** [comment]
That is the latest source code for version 18.0. I am going to push an update to 18.1 later today. But generally, the code on GitHub is the code we shipped with the latest public release.
